index.html

```diff
@@ -46,46 +46,257 @@ <h3 class="subtitle">
+              <div class="cards">
+                <div class="card">
+                    <div class="project-details">
+                        <div class="project-content">
+                            <h2 class="project-name">Professional Art Printing Data</h2>
+                            <p class="description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                has been the industry's standard
+                            </p>
+                            <ul class="boxes">
+                                <li><a href="#"><img src="images/tags/Tag5.png" alt="html"></a></li>
+                                <li><a href="#"><img src="images/tags/Tag6.png" alt="css"></a></li>
+                                <li><a href="#"><img src="images/tags/Tag7.png" alt="ruby"></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <a href="#" class="button see-project large">
+                        See Project
+                    </a>
+                </div>
+              </div>
+      
+    </section>
+    <section class="about">
+      <h1 class="titl">About me</h1>
+      <p class="about-me">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class= "res" href="#"><img src="images/resume.png" alt="resume"></a>
+
+        <img class= "about-ill" src="images/illustration/illustration-about-me-mobile.svg" alt="about">
+
+        <div class="border">
+          <img src="images/Vector4.png" alt="line">
+        </div>
+
+        <div class="language">
+          <img src="images/Rectangle55.png" alt="rectangle">
+          <h2 class="lang-header">Language</h2>
+          
+        </div>
+        <div class="skills">
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="language">
+          <img src="images/Rectangle56.png" alt="rectangle">
+          <h2 class="lang-header">Frameworks</h2>
+          
+        </div>
+        <div class="skills">
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="language">
+          <img src="images/Rectangle57.png" alt="rectangle">
+          <h2 class="lang-header">Skills</h2>
+          
+        </div>
+        <div class="skills">
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      
+
     </section>
```

Comment:
Great job, good use of sematic

- [ ] Kindly change the `h1` tag to any other heading tag because `h1` is used only for the main heading of your website for SEO reasons.

Reply:
i have used h2 tag instead